libpoolprotocols/PoolManager.cpp

```diff
@@ -155,7 +157,7 @@ void PoolManager::setClientHandlers()
 
         // Should not happen !
         if (!wp)
-            return;
+            return; 
```

Comment:
Unnecessary space added.

Reply:
fixed in https://github.com/ethereum-mining/ethminer/pull/1824/commits/eb95ad3ff0c00561f54b881424141c031a677da2